docs/_includes/api/replication.html

```diff
@@ -39,12 +39,12 @@
   retry: true
 }).on('change', function (info) {
   // handle change
-}).on('paused', function (err) {
+}).on('paused', function () {
   // replication paused (e.g. replication up to date, user went offline)
```

Comment:
We should add `(err)` to the documentation of both the paused events, it is passed in the event that the replication was paused due to an error